working-groups/README.md

```diff
@@ -1,7 +1,47 @@
 # Working Groups
 
-Each AMP Working Group is a segment of the community with knowledge/interest in specific area of AMP, with a mandate provided by the Technical Steering Committee.  See [GOVERNANCE.md](../GOVERNANCE.md#working-groups) for more details.
 
-## List of working groups
+An AMP [Working Group](https://github.com/ampproject/meta/blob/master/GOVERNANCE.md#working-groups) is a segment of the community with knowledge/interest in specific area of AMP.  Working Groups are created by AMP’s [Technical Steering Committee](https://github.com/ampproject/meta/blob/master/GOVERNANCE.md#technical-steering-committee-tsc). 
+
+AMP’s Working Groups are:
+
+* **Access control and subscriptions.**
```

Comment:
Consider replace this list with a search query like `wg-` into the repository view, so that this auto-updates to the list of active groups. (Or use the project topic feature)

Reply:
For the search query, would this just be a link that people would click to see the repos, e.g. like https://github.com/ampproject?utf8=%E2%9C%93&q=meta&type=&language=

I can do this once we get the repos set up, assuming we're going with per-WG repos.